# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dhcpclient/install.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- dhcpclient-snap_20071104/Makefile.vanilla	2008-07-30 22:01:58.000000000 +0000
+++ dhcpclient-snap_20071104/Makefile	2008-07-30 22:02:10.000000000 +0000
@@ -28,9 +28,9 @@
 
 .PHONY: install
 install: dhcpclient dhcpclient.$(MANEXT) dhcpclient-script.sample
-	install -m 755 -t $(DESTDIR)$(PREFIX)/sbin dhcpclient
-	install -m 755 -t $(DESTDIR)$(PREFIX)/sbin dhcpclient-script.sample
-	install -m 644 -t $(DESTDIR)$(MANDIR)/man$(MANEXT) dhcpclient.$(MANEXT)
+	install -m 755 dhcpclient $(DESTDIR)$(PREFIX)/sbin
+	install -m 755 dhcpclient-script.sample $(DESTDIR)$(PREFIX)/sbin
+	install -m 644 dhcpclient.$(MANEXT) $(DESTDIR)$(MANDIR)/man$(MANEXT)
 
 .depend: Makefile
 	touch .depend
